packages/oas-extensions/package.json

```diff
@@ -13,9 +13,9 @@
   "license": "ISC",
   "repository": "https://github.com/readmeio/api-explorer/tree/master/packages/oas-extensions",
   "devDependencies": {
-    "eslint": "^5.3.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-plugin-import": "^2.17.2",
+    "eslint": "^6.5.0",
```

Comment:
I think in lerna, they encourage moving shared devdeps up to the top level and using them from there. This may not work for executables though.

Reply:
It should work, or at least it has for the various Babel and Webpack plugins I've tried adding in the past!

Reply:
Oh good to know. I'll see if I can pull them out of here these individual packages and clean those lock files up.